5_People/personal-finances.md

```diff
@@ -14,9 +14,9 @@ After you know your numbers, you can do the next steps.
 
 ### Emergency fund and loans
 
-Investing is risky, and there can be years where the return is negative. That is also the worst time to sell your investments. Therefore you should always keep a 6-month emergency fund.
+Investing is risky, and there can be years where the return is negative. If you invested at the top before the 2008 crisis, you lost 50% by the beginning of 2009 and needed at least four years to get back to zero. That is also the worst time to sell your investments. Therefore you should always keep a 6-month emergency fund.
 
-There is an easy 5-8% return per year for many people with zero risks. What is it? Paying off your (short-term) loans. Save money and pay for furniture, car, or any other large purchase in cash.
+There is an easy risk-free 5-8% return per year for many people. What is it? Paying off your (short-term) loans. Save money and pay for furniture, car, or any other large purchase in cash.
```

Comment:
No return is risk-free. You get return precisely because you are taking on risk. 

`low-risk` is better